indexd/index/blueprint.py

```diff
@@ -452,8 +452,8 @@ def get_all_index_record_versions(record):
     """
     Get all record versions
     """
-    not_deleted = flask.request.args.get("not_deleted", "true").lower() == "true"
-    ret = blueprint.index_driver.get_all_versions(record, not_deleted=not_deleted)
+    exclude_deleted = flask.request.args.get("exclude_deleted", "false").lower() == "true"
```

Comment:
I think we should remove 't' as true in the other location if that is the case. That seems like an unnecessary shorthand.

Reply:
I agree that we should remove `t` as an option for `exclude_deleted` of `/bulk/documents/latest endpoint`. Though it will still be inconsistent with other query params like `skip_null` and `versioned` query params of other endpoints but we wouldn’t want to introduce a breaking change for this kind of consistency.